app/api/chat/route.ts

```
@@ -19,27 +20,32 @@ export async function POST(req: NextRequest) {
 
     let aiModel;
     switch (modelConfig.provider) {
-      case 'google':
+      case "google":
         const google = createGoogleGenerativeAI({ apiKey });
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)